copy_encrypted_ami.sh

```diff
@@ -210,7 +210,9 @@ echo -e "${COLOR}AMI created succesfully in the destination account:${NC} ${CREA
 # Copy Tags
 if [ "${TAG_OPT}x" != "x" ]; then
     AMI_TAGS=$(echo ${AMI_DETAILS} | jq '.Tags')"}"
-    NEW_AMI_TAGS="{\"Tags\":"$(echo ${AMI_TAGS} | tr -d ' ')
-    $(aws ec2 create-tags --resources ${CREATED_AMI} --cli-input-json ${NEW_AMI_TAGS} --profile ${DST_PROFILE} --region ${DST_REGION} || die "Unable to add tags to the AMI in the destination account. Aborting.")
-    echo -e "${COLOR}Tags added sucessfully${NC}"
+    if [ "${AMI_TAGS}x" != "x" ]; then
```

Comment:
If the user has used "-t" but the source has no tags, this block should not execute at all. As the response from line 22 (if you remove the "}" bit ) will be null, I would change that this line here to:
if [ "${AMI_TAGS}" != "null" ]; then

Reply:
Hmm, you're right we do get a null - but the `}` is there in the original file before my modifications and exists to ensure the later tagging operation has valid JSON (though it's a bit clunky IMHO)